--- a/pdf2zh_next/gui.py
+++ b/pdf2zh_next/gui.py
@@ -4132,7 +4132,7 @@
     if not auth_file or not user_list:
         try:
             demo.launch(
-                server_name="0.0.0.0",
+                server_name="127.0.0.1",
                 debug=True,
                 inbrowser=inbrowser,
                 share=share,
@@ -4159,7 +4159,7 @@
                 demo.launch(
                     debug=True,
                     inbrowser=inbrowser,
-                    share=True,
+                    share=False,
                     server_port=server_port,
                     allowed_paths=[str(p) for p in pdf_preview_allowed_paths],
                 )
@@ -4166,7 +4166,7 @@
     else:
         try:
             demo.launch(
-                server_name="0.0.0.0",
+                server_name="127.0.0.1",
                 debug=True,
                 inbrowser=inbrowser,
                 share=share,
@@ -4197,7 +4197,7 @@
                 demo.launch(
                     debug=True,
                     inbrowser=inbrowser,
-                    share=True,
+                    share=False,
                     auth=user_list,
                     auth_message=html,
                     server_port=server_port,
